scripts/scr_boundarytransition.gml

```diff
@@ -0,0 +1,7 @@
+boundarydoor = collision_rectangle(x,y,x+64,y+128,obj_boundarydoor,0,1);
```

Comment:
please avoid the use of 'magic numbers'. I believe you are referring to `sprite_width` and `sprite_height` here?